tests/unit/sagemaker/image_uris/test_algos.py

```diff
@@ -177,7 +177,8 @@ def test_algo_uris(algo):
 
     for region in regions.regions():
         uri = image_uris.retrieve(algo, region)
-        assert expected_uris.algo_uri(algo, accounts[region], region) == uri
+        version = "latest" if algo == "blazingtext" else 1
```

Comment:
should be a `str`?

```suggestion
        version = "latest" if algo == "blazingtext" else "1"
```

Reply:
To reiterate above point: nothing special in BlazingText wrt to versioning. It should be same as other 1P algos